src/components/cycle-section/cycle-section/CycleTopContainer.js

```diff
@@ -0,0 +1,19 @@
+import styles from './CycleSection.module.scss';
+import { Button } from 'primereact/Button';
+
+export const CycleTopContainer = ({ cycle }) => {
+  return (
+    <div className={styles.topContainer}>
+      <div>
+        <h2>Ciclo {cycle.order_number}</h2>
+        <h1>{cycle.name}</h1>
+      </div>
+      <div>
+        <Button
+          className={styles.cycleCompletitionButton}
```

Comment:
I think this class could be deleted, we can work with the same style in all the app. Deleting this class, the button will be as the primary buttons.

Reply:
Done!